ec2/ec2-create-instance.sh

```diff
@@ -23,10 +24,11 @@ usage() {
   echo "default AWS size is t2.xlarge to avoid OoM killer during integration tests (otherwise, t2.large should be fine)"
   echo "local log path will rsync Payara, Jacoco, Maven and other logs back to the specified path"
   echo "-d will destroy ("terminate") the AWS instance once testing and reporting completes"
+  echo "-v increases Ansible output verbosity"
   exit 1
 }
 
-while getopts ":a:r:b:g:p:i:s:t:l:d" o; do
+while getopts ":a:r:b:g:p:i:s:t:l:dv" o; do
```

Comment:
Is there supposed to be a colon between d and v like d:v?

Reply:
the colon determines whether an argument accepts a variable. in this case -d nor -v take an argument.